src/views/Vote/Input.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
```

Comment:
If the create proposal component uses more components, create a component folder to house the components.

Reply:
Done